README.md

```diff
@@ -1 +1,170 @@
-# Hello-world
\ No newline at end of file
+# Hello world
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  
+
+  <h3><b>hello world</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello world] <a name="about-project"></a>
+
+
+
+**[Hello world]** is the first project at microverse with tests on gitflow and linter errors.
+
+- [<a href="https://www.loom.com/share/3f4171ede5b34ca08a62a638f40fc4bc" target="_blank">Project video</a>](#)
+
+## 🛠 Built With <a name="sheldon wafula"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://index.html">HTML</a></li>
+  </ul>
+</details>
+
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="Heading"></a>
+
+
+- **[Heading]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="Now yet live"></a>
+
+- [<a href="#" target="_blank">Demo</a>](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+
+Clone this repository to your desired folder:
+
+
+
```

Comment:
- [ ] I have noticed that the section headings of the getting started section have been removed for example install and setup. Kindly try as much as possible to customize them as they are necessary for this project to start running ☺️.